astropy/table/operations.py

```diff
@@ -35,21 +35,24 @@ def _merge_table_meta(out, tables, metadata_conflicts='warn'):
 
 def _get_list_of_tables(tables):
     """
-    Check that tables is a Table or sequence of Tables.  Returns the
-    corresponding list of Tables.
+    Check that tables is a Table or sequence of Tables, Columns and/or rows. Convert each Row and
+    Column in the sequence to the corresponding Table objects and finally return the sequence of Tables.
     """
-    from .table import Table, Row
+    from .table import Table, Row, Column
 
     # Make sure we have a list of things
     if not isinstance(tables, collections.Sequence):
         tables = [tables]
 
-    # Make sure each thing is a Table or Row
-    if any(not isinstance(x, (Table, Row)) for x in tables) or len(tables) == 0:
-        raise TypeError('`tables` arg must be a Table or sequence of Tables or Rows')
+    # Make sure each thing is a Table or Row or Column
+    if any(not isinstance(x, (Table, Row, Column)) for x in tables) or len(tables) == 0:
+        raise TypeError('`tables` arg must be a Table or sequence of Tables or Rows or Column')
 
     # Convert any Rows to Tables
-    tables = [(x if isinstance(x, Table) else Table(x)) for x in tables]
+    tables = [(Table(x) if isinstance(x, Row) else x) for x in tables]
+
+    # Convert any Columns to Tables
+    tables = [(Table([x]) if isinstance(x, Column) else x) for x in tables]
```

Comment:
In order to allow for mixin columns this needs to be a little more complicated.
```
from .table import Table, Row, has_info_class, BaseColumnClass
```
Then the test is not `isinstance(x, Column)` but `has_info_class(x, BaseColumnClass)`.

Reply:
hi.. I am checking this.. and will make the appropriate changes.